drivers/hmis/app/models/hmis/ce/client_proxy.rb

```diff
@@ -24,8 +24,76 @@ class ClientProxy < GrdaWarehouseBase
 
     scope :for_warehouse_clients, -> { where(client_type: GrdaWarehouse::Hud::Client.sti_name) }
 
+    scope :matching_search_term, ->(search_term) do
+      search_term.strip!
```

Comment:
suggest not mutating the params
```suggestion
      search_term  = search_term.strip
```